Tests/OktaLoggerTests/Analytics/AnalyticsTests.swift

```diff
@@ -0,0 +1,9 @@
+//
+//  AnalyticsTests.swift
+//  OktaLoggerTests
+//
+//  Created by SP on 9/27/22.
+//  Copyright © 2022 Okta, Inc. All rights reserved.
+//
+
+import Foundation
```

Comment:
Tests?

Reply:
Have  some issues on test target
```swift
Undefined symbols for architecture x86_64:
  "type metadata accessor for OktaLoggerAnalytics.AppCenterAnalyticsProvider", referenced from:
      OktaLoggerTests.AnalyticsTests.testAnalytics() -> () in AnalyticsTests.o
  "OktaLoggerAnalytics.AppCenterAnalyticsProvider.__allocating_init(name: Swift.String, logger: OktaLogger.OktaLoggerProtocol?, appCenter: __C.MSACAnalytics.Type) -> OktaLoggerAnalytics.AppCenterAnalyticsProvider", referenced from:
      OktaLoggerTests.AnalyticsTests.testAnalytics() -> () in AnalyticsTests.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
``` 

Will add tests later if i cannot able to figure out the issue